zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -65,15 +66,15 @@ public Note getHomeNote(ServiceContext context,
                           ServiceCallback<Note> callback) throws IOException {
     String noteId = notebook.getConf().getString(ZEPPELIN_NOTEBOOK_HOMESCREEN);
     Note note = null;
-    if (noteId != null) {
+    if (!StringUtils.isBlank(noteId)) {
       note = notebook.getNote(noteId);
       if (note != null) {
         if (!checkPermission(noteId, Permission.READER, Message.OP.GET_HOME_NOTE, context,
             callback)) {
           return null;
         }
       } else {
-        callback.onFailure(new Exception("configured HomePage is not existed"), context);
+        callback.onFailure(new Exception("The configured home screen does not exist."), context);
```

Comment:
Renaming "HomePage" to "home screen" as the name in config is "zeppelin.notebook.homescreen" (https://github.com/apache/zeppelin/blob/master/conf/zeppelin-site.xml.template#L59)